components/espace-agent-components/documents/bilans-walled.tsx

```diff
@@ -15,7 +15,6 @@ const BilansSection: React.FC<{
         title="Bilans"
         id="bilans"
         uniteLegale={uniteLegale}
-        condition={!hasRights(session, EScope.bilansRne)}
```

Comment:
I dont get how you deal with bilans rights know that you removed the condition ?

Reply:
nevermind it was already useless